### ISSUE TITLE
bracken est_abundance.py: No such file or directory

### ISSUE THREAD

**Issue by nick-youngblut** [bioconda/bioconda-recipes#11629]
@dfornika I'm running `bracken  2.2   py36h2d50403_0   bioconda` on Ubuntu 18.04.1, and I'm getting the following error:

```
 >> Checking for Valid Options...
 >> Running Bracken
      >> python src/est_abundance.py -i tests/output_amy/kraken/36_T10.kreport -o tests/output_amy/kr
aken/36_T10.bracken -k /ebio/abt3_projects/databases/Kraken/K2_Progenomes/Kraken/database150mers.kmer
_distrib -l s -t 10
python: can't open file '/ebio/abt3_projects/software/dev/llmgp/.snakemake/conda/afe44be5/bin/src/est
_abundance.py': [Errno 2] No such file or directory
```

It seems that `bracken` is looking for `est_abundance.py` in bin/src/, but `est_abundance.py` is located in bin/.

**Comment by dfornika** [comment]
Thanks for reporting. Maybe we can piggy-back a fix for this into #11700 ?

**Comment by dfornika** [comment]
I believe this was fixed in #11700 . @nick-youngblut can you confirm and close this issue if that's the case?
